### DISCUSSION TITLE
st.experimental_rerun replaced

### DISCUSSION THREAD

**Discussion by ericgoldclub** [aahnik/tgcf#526] [Q&A]
Hello, I have a problem with the forwarder. It doesn't work and when I run the python code in Linux I get the message:
"st.experimental_rerun will be removed after 2024-04-01. Please replace st.experimental_rerun with st.rerun"
Does someone know how to fix this?

**Comment by Navneet-Choudhary** [comment]
Did you found any solution?

**Comment by scottnzuk** [comment]
I also have this issue on M1 MAC

**Comment by DevMd80** [comment]
same

**Comment by CorruptedMinds** [comment]
Try downgrading streamlit to version 1.26

**Comment by panikajoreserv** [comment]
Did you found any solution?
